package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@edx/frontend-component-footer",
-  "version": "1.0.0-semantically-released",
+  "name": "@edly-io/indigo-frontend-component-footer",
```

Comment:
Is this branch name still intended to be `edly-io/*`?

Reply:
I've reverted the branch name and made updates. Could you please review it?

An issue I want to highlight when executing npm run snapshot, where the tests fail due to the following error.
```
/Users/hina/mfes/frontend-component-footer/node_modules/@openedx/frontend-plugin-framework/node_modules/@edx/frontend-platform/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){export { modifyObjectKeys, camelCaseObject, snakeCaseObject, convertKeyNames, getQueryParameters, ensureDefinedConfig, parseURL, getPath } from './utils';
                                                                                             ^^^^^^

    SyntaxError: Unexpected token 'export'

       7 | import { Image } from '@edx/paragon';
       8 | import { getConfig } from '@edx/frontend-platform';
    >  9 | import { PluginSlot } from '@openedx/frontend-plugin-framework';
         | ^

```

Reply:
Agh, my guess is that Jest is not handling the `import` statement within `frontend-plugin-framework`'s Frontend Platform dependency for some reason. I've not seen this be an issue with other React repos, and I'm not sure what the best solution is for this, but I did notice that when `npm run test` is ran it shows this message: 

> Jest encountered an unexpected token
> 
>     This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.
> 
>     By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".
> 
>     Here's what you can do:
>      • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/en/ecmascript-modules for how to enable it.
>      • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
>      • If you need a custom transformation specify a "transform" option in your config.
>      • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.
> 
>     You'll find more details and examples of these config options in the docs:
>     https://jestjs.io/docs/en/configuration.html